src/routes/RemediationDetails.js

```diff
@@ -110,21 +112,25 @@ class RemediationDetails extends Component {
                 { value =>
                     value.permissions.read === false
                         ? <DeniedState/>
-                        : <React.Fragment>
-                            <PageHeader>
-                                <Breadcrumb>
-                                    <BreadcrumbItem>
-                                        <Link to='/'> Remediations </Link>
-                                    </BreadcrumbItem>
-                                    <BreadcrumbItem isActive> { remediation.name } </BreadcrumbItem>
-                                </Breadcrumb>
-                                <Level className="ins-c-level">
-                                    <LevelItem>
-                                        <PageHeaderTitle title={ remediation.name }/>
-                                    </LevelItem>
-                                    <LevelItem>
-                                        <Split gutter="md">
-                                            { this.state.isEntitled &&
+                        : <Switch>
+                            <Route exact path='/:id/:run_id' render={ () => (<ActivityDetails remediation={ remediation } issue={ {} } />) } />
```

Comment:
Is there a reason for the routes to be defined here instead of the central location https://github.com/RedHatInsights/insights-remediations-frontend/blob/master/src/Routes.js#L50